### DISCUSSION TITLE
How to make SPI build only the top-level `package.swift`

### DISCUSSION THREAD

**Discussion by tgrapperon** [SwiftPackageIndex/SwiftPackageIndex-Server#1531] [Q&A]
Hello!

I have a package at https://swiftpackageindex.com/tgrapperon/composable-effect-identifier

It has a main `package.swift` manifest at the root of the repository, and another one in some Example folder. This example `LonelyTimer` package depends on the repository's package.

When SPI tries to build the repository, it fails to build the `LonelyTimer` package, having trouble resolving the dependency. 
I initially depended on the local version with `.dependency(path: "../../")` and I've now switched to see if `.dependency(url:"the repo url like you would do regularly"` would help. If fails in both cases, with different errors (it finds none or two versions of the dependency!)

I'm using a package in the sample to demonstrate how modularity is preserved when using my project. It works fine when opened directly with Xcode, or using the workspace provided.

As a result, badges for the package are not representative.
 I would like to instruct SPI to only build the first, root-level `package.swift`, but I'm not finding the documentation for this file. Is this possible?.

Thanks a lot!

**Comment by finestructure** [comment]
Just realised this was a discussion, promoted to an issue here: https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/1532 :)
